plugins/org.jkiss.dbeaver.ext.hana/src/org/jkiss/dbeaver/ext/hana/model/data/HANAValueHandlerProvider.java

```diff
@@ -33,6 +33,8 @@ public class HANAValueHandlerProvider implements DBDValueHandlerProvider {
     public DBDValueHandler getValueHandler(DBPDataSource dataSource, DBDFormatSettings preferences,
             DBSTypedObject typedObject) {
         switch (typedObject.getTypeName()) {
+        case "REAL_VECTOR":
```

Comment:
You didn't add a constant here

Reply:
And in other places.

Reply:
@LonwoLonwo
Thanks for your feedback.

I did not use a constant here because there are hard-coded datatype names below.

I considered the following options:

1. Use the constant and make the code look inconsistent.
2. Introduce constants for `ST_Geometry` and `ST_Point`, which would add refactorings not related to this feature.
3. Use the hard-coded datatype name.

I opted for 3 but can change it. What's your proposal?

Reply:
Please also add constants for ST_Geometry and ST_Point. 
(2)